cmd/dns/cmd.go

```diff
@@ -50,7 +50,8 @@ func NewCmdValidateDns() *cobra.Command {
 
 			logger.Warn(ctx, "Using region: %s", config.region)
 			creds := credentials.NewStaticCredentialsProvider(os.Getenv("AWS_ACCESS_KEY_ID"), os.Getenv("AWS_SECRET_ACCESS_KEY"), os.Getenv("AWS_SESSION_TOKEN"))
-			cli, err := cloudclient.NewClient(ctx, logger, creds, config.region, "", nil)
+			// The use of t3.micro here is arbitrary; we just need to provide any valid machine type
+			cli, err := cloudclient.NewClient(ctx, logger, creds, config.region, "t3.micro", nil)
```

Comment:
This is out of scope of your PR, but we should really remove this requirement... its a bit silly.

Reply:
Completely agree. Tried to leave this field blank, but it refused to instantiate unless a valid type was provided

Reply:
I can work on a separate PR